cloud/huawei/clients/machineservice.go

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package clients
+
+import (
+	"github.com/gophercloud/gophercloud"
+	"github.com/gophercloud/gophercloud/openstack"
+)
+
+const (
+	HttpsHead = "https://"
+)
+
+// TODO: finish machine service
+type MachineService struct {
+	cloudserver *gophercloud.ServiceClient
+	region      string
+	version     string
+	apiUrl      string
+}
+
+// url to create client
```

Comment:
Plus, I would suggest comment of the public method should start with, for example, "NewMachineService ......" in case we would introduce golint check.